lib/provider/ui/windows/welcome/index.tsx

```diff
@@ -247,39 +293,55 @@ export function Welcome(): ReactNode {
   return (
-    <Window window="Welcome" defaultWidth={720} defaultHeight={454}>
-      <h1 css="flex-shrink: 0; font-size: 2rem; line-height: 2.625rem; font-weight: bold;">
+    <Window
+      window="Welcome"
+      defaultWidth={720}
+      defaultHeight={454}
+      className={cssModule.window}
+    >
+      <StyledTitle className={cssModule.title}>
         Welcome to @phuctm97
-      </h1>
-      <div css="flex-grow: 1; flex-shrink: 1; display: flex; overflow: hidden; margin-top: 10px;">
-        <StyledFrame variant="well">
+      </StyledTitle>
+      <ContentWindow className={cssModule.contentWindowWrapper}>
+        <StyledFrame variant="well" className={cssModule.contentWrapperFrame}>
           <Content tab={tab} />
         </StyledFrame>
-        <div css="flex-shrink: 0; display: flex; flex-direction: column; align-items: stretch;">
-          <Button css="flex-shrink: 0;" onClick={handleClickAboutWebsite}>
-            About this Website
-          </Button>
-          <Button
-            css="flex-shrink: 0; margin-top: 10px;"
-            onClick={handleClickAboutAuthor}
-          >
-            About the Author
-          </Button>
-          <Separator css="flex-shrink: 0; margin-top: 40px; margin-bottom: 20px;" />
-          <Button css="flex-shrink: 0;" onClick={handleClickAcknowledgements}>
-            Acknowledgements
-          </Button>
-          <Button css="flex-shrink: 0; margin-top: 10px;" onClick={openGithub}>
-            GitHub ↗
-          </Button>
-          <Button
-            css="flex-shrink: 0; margin-top: 10px;"
-            onClick={openXOrTwitter}
-          >
-            X/Twitter ↗
-          </Button>
-        </div>
-      </div>
+        <ActionButtonsContainer className={cssModule.actionButtonsContainer}>
+          <ButtonGroup className={cssModule.buttonGroup}>
+            <Button css="flex-shrink: 0;" onClick={handleClickAboutWebsite}>
+              <span>About this Website</span>
+              <Confcp102 />
+            </Button>
+            <Button
+              css="flex-shrink: 0; margin-top: 10px;"
+              className={cssModule.button}
+              onClick={handleClickAboutAuthor}
+            >
+              <span>About the Author</span>
+              <Ulclient1002 />
+            </Button>
+            <StyledSeparator className={cssModule.separator} />
+            <Button css="flex-shrink: 0;" onClick={handleClickAcknowledgements}>
+              <span>Acknowledgements</span>
+              <Bookmark />
+            </Button>
+          </ButtonGroup>
+          <ButtonGroup className={cssModule.buttonGroup}>
+            <Button
+              css="flex-shrink: 0; margin-top: 10px;"
+              onClick={openGithub}
+            >
+              GitHub ↗
+            </Button>
+            <Button
+              css="flex-shrink: 0; margin-top: 10px;"
+              onClick={openXOrTwitter}
+            >
+              X/Twitter ↗
+            </Button>
```

Comment:
I think we should hide the ` ↗` on mobile to leave more space for smaller mobile screens.